WordPress/src/main/res/layout/reader_cardview_post_new.xml

```diff
@@ -191,7 +191,7 @@
             android:text="@string/like"
             app:drawableStartCompat="@drawable/ic_like_new_selector"
             app:layout_goneMarginStart="0dp"
-            app:layout_constraintWidth_max="80dp"
+            app:layout_constraintWidth_max="140dp"
```

Comment:
I just noticed the `like` button is the only one with a `constraintWidth_max` property on this file. I don't remember now if that's on purpose or if I just copied the code from somewhere (likely `reader_include_post_detail_footer_new.xml`) and forgot to delete the property for this one view.

Did you try removing this property and checking if things work as expected? Maybe we don't need a max width for these items since we already have them on a chain and `constrainedWidth="true"` on all of them, which should already take care of longer strings and avoid overflowing out of the screen.

Reply:
Yes, removing it was the first thing I tried and without it this is what happens with a longer "Like" text:

<img width="393" alt="image" src="https://github.com/wordpress-mobile/WordPress-Android/assets/14964993/219ffe3a-8dcc-4cfe-bdbf-3c491199cca4">

The same problem happens if the "Reblog" or "Comment" action labels are too big, but it seemed out of scope for this issue.

Here's how it looks like with the width max property:

<img width="382" alt="image" src="https://github.com/wordpress-mobile/WordPress-Android/assets/14964993/e3ca5466-5b6c-4640-b9fe-e28d95a20705">

Reply:
Ah, got it. So the `constrainedWidth` is not enough to avoid this, so in this case I guess the `constraintWidth_max` makes sense! Thanks!